.env

```diff
@@ -0,0 +1,3 @@
+NEXT_PUBLIC_AUTH0_DOMAIN=maffin-dev.eu.auth0.com
+NEXT_PUBLIC_AUTH0_CLIENT_ID=mMmnR4NbQOnim9B8QZfe9wfFuaKb8rwW
+NEXT_PUBLIC_AUTH0_SCOPES=profile email
```

Comment:
This is a valid default file for configuring nextjs according to https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables#environment-variable-load-order.

All the other config files are not "valid" but we copy them in the build to `.env` depending on the user branch